charts/hivemq-platform/values.yaml

```diff
@@ -68,6 +68,12 @@ nodes:
   #        key: my-configmap-key
   #        optional: true
```

Comment:
Also can you move the containerSecurity section right under the podSecuritySection instead of having env section inbetween?